utility/src/main/java/org/oppia/android/util/logging/firebase/BUILD.bazel

```diff
@@ -0,0 +1,79 @@
+"""
+General purpose utilities for firebase logging.
+"""
+
+load("@dagger//:workspace_defs.bzl", "dagger_rules")
+load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kt_android_library")
+
+kt_android_library(
```

Comment:
I think we can omit the firebase_ prefix from all the libraries in the BUILD file. So they can be declared as

`firebase:log_reporting_module`

instead of

`firebase:firebase_log_reporting_module`

Reply:
Ok @fsharpasharp. Done.